vips/vip-407/bscmainnet.ts

```diff
@@ -0,0 +1,247 @@
+import { parseUnits } from "ethers/lib/utils";
+import { ethers } from "hardhat";
+import { NETWORK_ADDRESSES } from "src/networkAddresses";
+import { LzChainId, ProposalType } from "src/types";
+import { makeProposal } from "src/utils";
+
+const { ethereum } = NETWORK_ADDRESSES;
+
+export const ACM = "0x230058da2D23eb8836EC5DB7037ef7250c56E25E";
+export const COMPTROLLER_ETHENA = "0x562d2b6FF1dbf5f63E233662416782318cC081E4";
+
+export const PT_USDe_27MAR2025 = "0x8A47b431A7D947c6a3ED6E42d501803615a97EAa";
+export const PT_sUSDE_27MAR2025 = "0xE00bd3Df25fb187d6ABBB620b3dfd19839947b81";
+export const sUSDe = "0x9D39A5DE30e57443BfF2A8307A4256c8797A3497";
+export const USDC = "0xA0b86991c6218b36c1d19D4a2e9Eb0cE3606eB48";
+const USDe = "0x4c9edd5852cd905f086c759e8383e09bff1e68b3";
+export const VTOKEN_RECEIVER = "0x3e8734ec146c981e3ed1f6b582d447dde701d90c";
+export const PRIME = "0x14C4525f47A7f7C984474979c57a2Dccb8EACB39";
+
+export const VPT_USDe_27MAR2025_ETHENA = "0x62D9E2010Cff87Bae05B91d5E04605ef864ABc3B";
+export const VPT_sUSDE_27MAR2025_ETHENA = "0xCca202a95E8096315E3F19E46e19E1b326634889";
+
+const sUSDE_ERC4626ORACLE = "0x67841858BCCA8dF50B962d6A314722a6AEC0970e";
+const PendleOracle_PT_USDe_27MAR2025 = "0x721C02F98bE5ef916F6574E53700a25473742093";
+const PendleOracle_PT_sUSDe_27MAR2025 = "0x17B49f36878c401C1fE4D7Bf6D9CeBAAFBf4edE2";
+const BOUND_VALIDATOR = "0x1Cd5f336A1d28Dff445619CC63d3A0329B4d8a58";
+
+const REDSTONE_USDe_FEED = "0xbC5FBcf58CeAEa19D523aBc76515b9AEFb5cfd58";
+const CHAINLINK_USDe_FEED = "0xa569d910839Ae8865Da8F8e70FfFb0cBA869F961";
+
+export const EXCHANGE_RATE = parseUnits("1.1", 18);
```

Comment:
This shouldn't be needed for mainnet

Reply:
[1866d37](https://github.com/VenusProtocol/vips/pull/440/commits/1866d377e77845c38ab5502842a9fee5915bfbb9)